.github/workflows/e2e-tests.yml

```diff
@@ -14,5 +14,5 @@ jobs:
         if: ${{ failure() }}
         uses: actions/upload-artifact@v2
         with:
-          name: e2e-test-output-${{ github.sha }}-${{ github.run_id }}
+          name: e2e-test-output-${{ github.event.pull_request.head.sha }}-${{ github.run_id }}
```

Comment:
is `github.event.pull_request.head` available when this workflow is run against `master`?

Reply:
do you mean when it is a commit instead of a PR? 
We use PR for commenting purpose. Gather all the commits for a PR (by commit SHAs) and comment on the PR. If it is a commit to the master without PR, the result should be counted towards periodic report instead of PR comment. So it should be fine right? wdyt. Thanks

Reply:
I am not entirely understanding the question here because this PR is opened against the master.

Reply:
oh, I get it now, you are saying if we run this test without a PR what would happen. I have not tried it. I'll check